### PR TITLE
chore: Use log.Default().Print* functions instead of fmt.Print*

### PR DESCRIPTION
**Pull request by asalkeld**
This will make it easier for the cli to capture the output.

### CODE REVIEW

File: pkg/membrane/membrane.go [hunk 1]
```diff
@@ -141,15 +143,15 @@ func (s *Membrane) startChildProcess() error {
...
+	log.Default().Println("Starting Child Process")
 	childProcess := exec.Command(s.childCommand[0], s.childCommand[1:]...)
 	childProcess.Stdout = os.Stdout
 	childProcess.Stderr = os.Stderr
 	applicationError := childProcess.Start()
 
 	// Actual panic here, we don't want to start if our userland code cannot successfully start
 	if applicationError != nil {
-		return fmt.Errorf("There was an error starting the child process: %v", applicationError)
+		return fmt.Errorf("there was an error starting the child process: %w", applicationError)
```
Comment (HomelessDinosaur) [inline pkg/membrane/membrane.go:154]: Could this be a log.Default().Fatal if its an actual panic?

File: pkg/membrane/membrane.go [hunk 2]
```diff
@@ -191,7 +193,7 @@ func (s *Membrane) Start() error {
 	}
 	lis, err := net.Listen("tcp", s.serviceAddress)
 	if err != nil {
-		return fmt.Errorf("Could not listen on configured service address: %v", err)
+		return fmt.Errorf("could not listen on configured service address: %w", err)
```
Comment (HomelessDinosaur) [inline pkg/membrane/membrane.go:196]: Could this be log.Default().Printf?

Comment (asalkeld) [reply pkg/membrane/membrane.go:196]: it's returning an error. that should bubble up to a log at some point.

File: pkg/membrane/membrane.go (hunk 1 above)
Comment (asalkeld) [reply pkg/membrane/membrane.go:154]: no, it will return  an error to the caller. We shouldn't assume failure handling

**Review by HomelessDinosaur — APPROVED** [review]

**Review by tjholm — APPROVED** [review]

**Pull request merged**